src/System.Collections.NonGeneric/src/System/Collections/Comparer.cs

```diff
@@ -16,14 +16,23 @@
 
 namespace System.Collections
 {
+    /// <summary>Compares two objects for equivalence, where string comparisons are case-sensitive.</summary>
+    /// <filterpriority>2</filterpriority>
     public sealed class Comparer : IComparer
     {
         private CompareInfo _compareInfo;
+        /// <summary>Represents an instance of <see cref="T:System.Collections.Comparer" /> that is associated with the <see cref="P:System.Threading.Thread.CurrentCulture" /> of the current thread. This field is read-only.</summary>
+        /// <filterpriority>1</filterpriority>
         public static readonly Comparer Default = new Comparer(CultureInfo.CurrentCulture);
+        /// <summary>Represents an instance of <see cref="T:System.Collections.Comparer" /> that is associated with <see cref="P:System.Globalization.CultureInfo.InvariantCulture" />. This field is read-only.</summary>
```

Comment:
Yeah, I prefer that as well.  Is there a guideline on spacing that we can try to get the formatter to adhere to?